echopype/tests/convert/test_convert_ek80.py

```diff
@@ -99,21 +99,25 @@ def test_convert_ek80_complex_matlab(ek80_path):
     # Convert file
     echodata = open_raw(raw_file=ek80_raw_path_bb, sonar_model='EK80')
 
+    # get indices of sorted frequency_nominal values. This is necessary
+    # because the frequency_nominal values are not always in ascending order.
+    sorted_freq_ind = np.argsort(echodata.beam.frequency_nominal)
+
     # check water_level
     assert (echodata["Platform"]["water_level"] == 0).all()
 
     # Test complex parsed data
     ds_matlab = loadmat(ek80_matlab_path_bb)
     assert np.array_equal(
-        echodata.beam.backscatter_r.isel(frequency=0, ping_time=0)
+        echodata.beam.backscatter_r.isel(channel=sorted_freq_ind[0], ping_time=0)
```

Comment:
Similar to the above it seems a good idea to change this to use `.sel(channel="SOME_CHANNEL")` here, for the sake of readability.